spec/models/spree/customer_return_spec.rb

```diff
@@ -0,0 +1,128 @@
+describe Spree::CustomerReturn, type: :model do
+
+  # => Actually, just in case you want to regenerate vcr you must manually
```

Comment:
This logic is very complex! Do you think can be created an helper to do these steps and avoid human errors?